src/clients/AwsProvisioner.js

```diff
@@ -5,7 +5,7 @@ import Client from '../Client';
 export default class AwsProvisioner extends Client {
   constructor(options = {}) {
     super({
-      baseUrl: 'https://aws-provisioner.taskcluster.net/v1',
+      baseUrl: 'https://staging-aws-provisioner.herokuapp.com/v1',
```

Comment:
This, however, is problematic!  It looks like the [staging provisioner has published to the production bucket](http://references.taskcluster.net/aws-provisioner/v1/exchanges.json).  @jhford do you want to take a look?